### ISSUE TITLE
Forum Activity delay when first opened

### ISSUE THREAD

**Issue by romanr** [uservoice/uservoice-android-sdk#11]
When Forum activity is opened First time (tested on 2.3, 4.x devices), It displays
"Post Idea, I suggest you___ " and blank screen under it for 2-3 seconds.

After 2-3 seconds spinner appears and it's visible that it loads forums.
When you open Forum 2nd time it starts loading forums right away.

This is a big UX problem because user will open this menu and will see that it only asks for suggestion and nothing is loading. And user will go back before seeing that it is loading something.
Basically this delay before loading makes this activity useless since it will never be discovered.
